tests/playwright/sanity/assets/dev/js/editor/elements/views/widget.test.js

```diff
@@ -52,14 +52,14 @@ test.describe( `$e.run( 'editor/elements/views/widget' )`, () => {
 		expect( emptyViewPlaceholderCount ).toBe( 1 );
 	} );
 
-	test.skip( 'Check if the empty placeholder is displayed inside the Basic Gallery widget', async ( { page }, testInfo ) => {
+	test( 'Check if the empty placeholder is displayed inside the Basic Gallery widget', async ( { page }, testInfo ) => {
 		// Arrange.
 		const wpAdmin = new WpAdminPage( page, testInfo ),
 			editor = await wpAdmin.useElementorCleanPost();
 
 		// Act.
 		await editor.addWidget( 'image-gallery' );
-		await editor.getPreviewFrame().waitForSelector( '.elementor-widget .elementor-widget-empty-icon' );
+		await editor.getPreviewFrame().waitForSelector( '.elementor-widget-image-gallery .elementor-widget-empty-icon' );
 
 		const emptyViewPlaceholderCount = await editor.getPreviewFrame().locator( '.elementor-widget-image-gallery .elementor-widget-empty-icon' ).count();
```

Comment:
About line 99:

Can we try to make this less flaky?

Could we use the 'waitForSelector', like on line 27: 

`await editor.getPreviewFrame().waitForSelector( '.elementor-widget .elementor-widget-empty-icon' );`